packages/cli/src/internal/options.ts

```diff
@@ -1428,10 +1428,16 @@ const wizardInternal = (self: Instruction, config: CliConfig.CliConfig): Effect.
         getHelpInternal(self.options as Instruction),
         InternalHelpDoc.sequence(defaultHelp)
       )
+      const defaultChoice = Option.isOption(self.fallback)
+        ? Option.match(self.fallback, {
+          onNone: () => ({ title: "None", description: "It has no value as default", value: true }),
+          onSome: (a) => ({ title: `Default [${JSON.stringify(a)}]`, value: true })
```

Comment:
@ricardo-valero - thanks for kick-starting this! 

Since the Wizard asks the user:

> This option is optional - use the default? ›

Perhaps to keep things a bit more semantic we could just make the options: 
- `Yes - use the default (<print default if available>)`
- `No - use a custom value`

What do you think?

Reply:
Yes that's better